bindings/python/iota_sdk/types/node_info.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import List, Optional
+from iota_sdk.types.common import HexStr
+
+
+@dataclass
+class NodeInfoMilestone:
+    """Milestone info.
+    """
+
+    index: int
+    timestamp: Optional[int] = None
+    milestoneId: Optional[HexStr] = None
+
+
+@dataclass
+class NodeInfoStatus:
+    """Node status.
+    """
+
+    isHealthy: bool
+    latestMilestone: NodeInfoMilestone
+    confirmedMilestone: NodeInfoMilestone
+    pruningIndex: int
+
+
+@dataclass
+class NodeInfoMetrics:
+    """Node metrics.
+    """
+
+    blocksPerSecond: float
+    referencedBlocksPerSecond: float
+    referencedRate: float
+
+
+@dataclass
+class RentStructure:
+    """Rent structure for the storage deposit
```

Comment:
```suggestion
    """Rent structure for the storage deposit.
```
:stuck_out_tongue_winking_eye: